### ISSUE TITLE
RF-gate doesn't work without RF source in digi

### ISSUE THREAD

**Issue by PhirePhly** [PhirePhly/aprx#7]
Per Hes, an I-gate only config doesn't add heard stations to the historydb, so implicit IS-to-RF gated packets get dropped as the stations haven't been "heard" on the RF interface.

**Comment by kd7lxl** [comment]
The pastebin has an expiration. Here it is in non-expiring form:
```
netax25rx packet len=42 from rx_socket;  family=17 protocol=200 ifindex=4 hatype=3 pkttype=0 halen=0
Received frame of 42 bytes from ax0: OH2RDK-2
erlang_add(OH2RDK-2, RX, 52, 1)
ax25_format_to_tnc() len=41 82 a0 96 60 60 66 60 9e|90 6e 98 b4 84 f0 ae 92|88 8a 64 40 63 03 f0 3a|55 54 46 2d 38 20 20 20|20 3a 74 65 73 74 7b 35|0d  = APK0030O|H7LZBxWI|DE2 c..:|UTF-8   | :test{5|.
13437298        OH2RDK-2  R     OH7LZB-8>APK003,WIDE2-1::UTF-8    :test{5
interface_receive_ax25() no receivers for source OH2RDK-2
telemetry_postpoll()  telemetrytime=1141s  labeltime=61s
telemetry_prepoll()
beacon_postpoll()
.. igate from aprsis
interface_receive_3rdparty() aif=0x8070300, aif->digicount=1
pbuf_alloc(31,26) -> 0xb7740000
.. parse_aprs() rc=OK  type=0x8008 srcif=APRSIS tnc2addr='UTF-8>APRS::OH7LZB-8 :ack5'  info_start=':OH7LZB-8 :ack5'
pbuf_free(0xb7740000)
historydb_lookup('OH7LZB-8 ') -> i=40 .. no match
.. parse_aprs() rc=OK  type=0x8008 srcif=APRSIS tnc2addr='UTF-8>APRS::OH7LZB-8 :ack5'  info_start=':OH7LZB-8 :ack5'
## produce 3rd-party AX.25 frames for transmit, and original source one for filtering:
ax25hdr 82 a0 a4 a6 40 40 60 9e|90 64 a4 88 96 65  = APRS  0O|H2RDKe
Formatted AX.25: OH2RDK-2>APRS:}UTF-8>APRS,TCPIP,OH2RDK-2*::OH7LZB-8 :ack5
 filter tnc2addr = UTF-8>APRS,TCPIP*,qAC,OH7LZB-UT
pbuf_alloc(59,75) -> 0xb7740000
historydb_lookup('OH7LZB-8 ') -> i=40 .. no match
.. parse_aprs() rc=OK  type=0x08 srcif=APRSIS tnc2addr='UTF-8>APRS,TCPIP*,qAC,OH7LZB-UT:}UTF-8>APRS,TCPIP,OH2RDK-2*::OH7LZB-8 :ack5'  info_start='}UTF-8>APRS,TCPIP,OH2RDK-2*::OH7LZB-8 :ack5'
## process source filter
 key='UTF-8' hash=15 .. inserting new history entry.
historydb_insert_heard(APRSIS) v=0xb7585000
```

**Comment by PhirePhly** [comment]
Fixed in commit f38cd0652afc11a08662568068f4965f01ce25cd
